com.hazelcast.mapstore.GenericMapStoreIntegrationTest.testClear (and its extensions) [HZ-2430]

_master_ (commit f1f6e5efe9ebf76a9ebc02b791d7bd77a4854426)

Failed on oracle-11 and Postgres 13: https://jenkins.hazelcast.com/view/Official%20Builds/job/Hazelcast-master-postgres-13/2/testReport/com.hazelcast.mapstore.postgres/PostgreGenericMapStoreIntegrationTest/testClear/
<details><summary>Stacktrace:</summary>

```
org.junit.runners.model.TestTimedOutException: test timed out after 30000 milliseconds
	at java.base@11.0.18/jdk.internal.misc.Unsafe.park(Native Method)
	at java.base@11.0.18/java.util.concurrent.locks.LockSupport.park(LockSupport.java:194)
	at java.base@11.0.18/java.util.concurrent.CompletableFuture$Signaller.block(CompletableFuture.java:1796)
	at java.base@11.0.18/java.util.concurrent.ForkJoinPool.managedBlock(ForkJoinPool.java:3128)
	at java.base@11.0.18/java.util.concurrent.CompletableFuture.waitingGet(CompletableFuture.java:1823)
	at java.base@11.0.18/java.util.concurrent.CompletableFuture.get(CompletableFuture.java:1998)
	at app//com.hazelcast.client.impl.proxy.ClientMapProxy.putAllInternal(ClientMapProxy.java:1663)
	at app//com.hazelcast.client.impl.proxy.ClientMapProxy.putAll(ClientMapProxy.java:1586)
	at app//com.hazelcast.mapstore.GenericMapStoreIntegrationTest.testClear(GenericMapStoreIntegrationTest.java:410)
	at java.base@11.0.18/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base@11.0.18/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base@11.0.18/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base@11.0.18/java.lang.reflect.Method.invoke(Method.java:566)
	at app//org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at app//org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at app//org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at app//org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at app//com.hazelcast.test.FailOnTimeoutStatement$CallableStatement.call(FailOnTimeoutStatement.java:115)
	at app//com.hazelcast.test.FailOnTimeoutStatement$CallableStatement.call(FailOnTimeoutStatement.java:107)
	at java.base@11.0.18/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base@11.0.18/java.lang.Thread.run(Thread.java:834)

```
</details>



Standard output can be found here - https://s3.console.aws.amazon.com/s3/buckets/j-artifacts/Hazelcast-master-postgres-13/2/

Reopening as issue occurred again on: _master_ (commit cee0f6eef8a56950e79e8534543790ff87a00476)
with OpenJDK11 (arm64): https://jenkins.hazelcast.com/job/Hazelcast-master-OpenJDK11-arm64-nightly/32/testReport/com.hazelcast.jet.sql_nightly/SqlSTSInnerNonEquiJoinFaultToleranceStressTest/stressTest_2/
<details><summary>Stacktrace:</summary>

```
org.opentest4j.MultipleFailuresError: 
Multiple Failures (2 failures)
	com.hazelcast.sql.HazelcastSqlException: Object 'source_topic_o_e31848e1_964c_44b9_a276_e6f77f3b3f63' not found, did you forget to CREATE MAPPING?
	java.lang.NullPointerException: <no message>
	at com.hazelcast.test.HazelcastSerialClassRunner.runChild(HazelcastSerialClassRunner.java:50)
	at com.hazelcast.test.HazelcastSerialClassRunner.runChild(HazelcastSerialClassRunner.java:29)
	at com.hazelcast.test.AfterClassesStatement.evaluate(AfterClassesStatement.java:41)
	at com.hazelcast.test.OverridePropertyRule$1.evaluate(OverridePropertyRule.java:66)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.lang.Thread.run(Thread.java:829)
	Suppressed: com.hazelcast.sql.HazelcastSqlException: Object 'source_topic_o_e31848e1_964c_44b9_a276_e6f77f3b3f63' not found, did you forget to CREATE MAPPING?
		at com.hazelcast.sql.impl.CoreQueryUtils.toPublicException(CoreQueryUtils.java:50)
		at com.hazelcast.sql.impl.SqlServiceImpl.execute(SqlServiceImpl.java:221)
		at com.hazelcast.sql.impl.SqlServiceImpl.execute(SqlServiceImpl.java:172)
		at com.hazelcast.sql.impl.SqlServiceImpl.execute(SqlServiceImpl.java:168)
		at com.hazelcast.sql.impl.SqlServiceImpl.execute(SqlServiceImpl.java:164)
		at com.hazelcast.sql.SqlService.execute(SqlService.java:89)
		at com.hazelcast.jet.sql_nightly.SqlSTSInnerEquiJoinFaultToleranceStressTest.setUp(SqlSTSInnerEquiJoinFaultToleranceStressTest.java:149)
		at java.base/java.lang.reflect.Method.invoke(Method.java:566)
		at com.hazelcast.test.AbstractHazelcastClassRunner$ThreadDumpAwareRunAfters.evaluate(AbstractHazelcastClassRunner.java:365)
		at com.hazelcast.test.DumpBuildInfoOnFailureRule$1.evaluate(DumpBuildInfoOnFailureRule.java:37)
		at com.hazelcast.test.jitter.JitterRule$1.evaluate(JitterRule.java:104)
		at com.hazelcast.test.metrics.MetricsRule$1.evaluate(MetricsRule.java:63)
		... 6 more
	Caused by: com.hazelcast.sql.impl.QueryException: Object 'source_topic_o_e31848e1_964c_44b9_a276_e6f77f3b3f63' not found, did you forget to CREATE MAPPING?
		at com.hazelcast.sql.impl.QueryException.error(QueryException.java:73)
		at com.hazelcast.jet.sql.impl.validate.HazelcastSqlValidator.newValidationError(HazelcastSqlValidator.java:542)
		at org.apache.calcite.sql.validate.IdentifierNamespace.resolveImpl(IdentifierNamespace.java:183)
		at org.apache.calcite.sql.validate.IdentifierNamespace.validateImpl(IdentifierNamespace.java:188)
		at org.apache.calcite.sql.validate.AbstractNamespace.validate(AbstractNamespace.java:88)
		at org.apache.calcite.sql.validate.SqlValidatorImpl.validateNamespace(SqlValidatorImpl.java:1144)
		at org.apache.calcite.sql.validate.SqlValidatorImpl.validateQuery(SqlValidatorImpl.java:1115)
		at com.hazelcast.jet.sql.impl.validate.HazelcastSqlValidator.validateQuery(HazelcastSqlValidator.java:183)
		at org.apache.calcite.sql.validate.SqlValidatorImpl.validateFrom(SqlValidatorImpl.java:3456)
		at org.apache.calcite.sql.validate.SqlValidatorImpl.validateFrom(SqlValidatorImpl.java:3435)
		at org.apache.calcite.sql.validate.SqlValidatorImpl.validateSelect(SqlValidatorImpl.java:3792)
		at org.apache.calcite.sql.validate.SelectNamespace.validateImpl(SelectNamespace.java:61)
		at org.apache.calcite.sql.validate.AbstractNamespace.validate(AbstractNamespace.java:88)
		at org.apache.calcite.sql.validate.SqlValidatorImpl.validateNamespace(SqlValidatorImpl.java:1144)
		at org.apache.calcite.sql.validate.AbstractNamespace.getRowType(AbstractNamespace.java:119)
		at org.apache.calcite.sql.validate.SelectNamespace.getRowType(SelectNamespace.java:32)
		at org.apache.calcite.sql.validate.AbstractNamespace.getType(AbstractNamespace.java:130)
		at org.apache.calcite.sql.validate.SelectNamespace.getType(SelectNamespace.java:32)
		at org.apache.calcite.sql.validate.SqlValidatorImpl.deriveType(SqlValidatorImpl.java:1899)
		at org.apache.calcite.sql.SqlOperator.deriveOperandType(SqlOperator.java:687)
		at org.apache.calcite.sql.SqlOperator.constructArgTypeList(SqlOperator.java:677)
		at org.apache.calcite.sql.SqlFunction.deriveType(SqlFunction.java:250)
		at org.apache.calcite.sql.SqlFunction.deriveType(SqlFunction.java:231)
		at org.apache.calcite.sql.validate.SqlValidatorImpl$DeriveTypeVisitor.visit(SqlValidatorImpl.java:6534)
		at org.apache.calcite.sql.validate.SqlValidatorImpl$DeriveTypeVisitor.visit(SqlValidatorImpl.java:6521)
		at org.apache.calcite.sql.SqlCall.accept(SqlCall.java:166)
		at org.apache.calcite.sql.validate.SqlValidatorImpl.deriveTypeImpl(SqlValidatorImpl.java:1916)
		at org.apache.calcite.sql.validate.SqlValidatorImplBridge.deriveTypeImpl(SqlValidatorImplBridge.java:40)
		at com.hazelcast.jet.sql.impl.validate.HazelcastSqlValidator.deriveTypeImpl(HazelcastSqlValidator.java:399)
		at org.apache.calcite.sql.validate.ProcedureNamespace.validateImpl(ProcedureNamespace.java:59)
		at org.apache.calcite.sql.validate.AbstractNamespace.validate(AbstractNamespace.java:88)
		at org.apache.calcite.sql.validate.SqlValidatorImpl.validateNamespace(SqlValidatorImpl.java:1144)
		at org.apache.calcite.sql.validate.SqlValidatorImpl.validateQuery(SqlValidatorImpl.java:1115)
		at com.hazelcast.jet.sql.impl.validate.HazelcastSqlValidator.validateQuery(HazelcastSqlValidator.java:183)
		at org.apache.calcite.sql.validate.SqlValidatorImpl.validateTableFunction(SqlValidatorImpl.java:3506)
		at org.apache.calcite.sql.validate.SqlValidatorImpl.validateFrom(SqlValidatorImpl.java:3453)
		at org.apache.calcite.sql.validate.SqlValidatorImpl.validateFrom(SqlValidatorImpl.java:3435)
		at org.apache.calcite.sql.validate.SqlValidatorImpl.validateSelect(SqlValidatorImpl.java:3792)
		at org.apache.calcite.sql.validate.SelectNamespace.validateImpl(SelectNamespace.java:61)
		at org.apache.calcite.sql.validate.AbstractNamespace.validate(AbstractNamespace.java:88)
		at org.apache.calcite.sql.validate.SqlValidatorImpl.validateNamespace(SqlValidatorImpl.java:1144)
		at org.apache.calcite.sql.validate.SqlValidatorImpl.validateQuery(SqlValidatorImpl.java:1115)
		at com.hazelcast.jet.sql.impl.validate.HazelcastSqlValidator.validateQuery(HazelcastSqlValidator.java:183)
		at org.apache.calcite.sql.SqlSelect.validate(SqlSelect.java:282)
		at org.apache.calcite.sql.validate.SqlValidatorImpl.validateScopedExpression(SqlValidatorImpl.java:1090)
		at org.apache.calcite.sql.validate.SqlValidatorImpl.validate(SqlValidatorImpl.java:796)
		at com.hazelcast.jet.sql.impl.validate.HazelcastSqlValidator.validate(HazelcastSqlValidator.java:178)
		at com.hazelcast.jet.sql.impl.parse.SqlCreateView.validate(SqlCreateView.java:104)
		at com.hazelcast.jet.sql.impl.validate.HazelcastSqlValidator.validate(HazelcastSqlValidator.java:146)
		at com.hazelcast.jet.sql.impl.parse.QueryParser.parse0(QueryParser.java:80)
		at com.hazelcast.jet.sql.impl.parse.QueryParser.parse(QueryParser.java:56)
		at com.hazelcast.jet.sql.impl.OptimizerContext.parse(OptimizerContext.java:157)
		at com.hazelcast.jet.sql.impl.CalciteSqlOptimizer.prepare(CalciteSqlOptimizer.java:325)
		at com.hazelcast.sql.impl.SqlServiceImpl.prepare(SqlServiceImpl.java:280)
		at com.hazelcast.sql.impl.SqlServiceImpl.query0(SqlServiceImpl.java:259)
		at com.hazelcast.sql.impl.SqlServiceImpl.execute(SqlServiceImpl.java:204)
		... 16 more
	Caused by: org.apache.calcite.runtime.CalciteContextException: From line 1, column 58 to line 1, column 108: Object 'source_topic_o_e31848e1_964c_44b9_a276_e6f77f3b3f63' not found, did you forget to CREATE MAPPING?
		at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)
		at org.apache.calcite.runtime.Resources$ExInstWithCause.ex(Resources.java:505)
		at org.apache.calcite.sql.SqlUtil.newContextException(SqlUtil.java:948)
		at org.apache.calcite.sql.SqlUtil.newContextException(SqlUtil.java:933)
		at com.hazelcast.jet.sql.impl.validate.HazelcastSqlValidator.newValidationError(HazelcastSqlValidator.java:535)
		... 70 more
	Caused by: org.apache.calcite.sql.validate.SqlValidatorException: Object 'source_topic_o_e31848e1_964c_44b9_a276_e6f77f3b3f63' not found, did you forget to CREATE MAPPING?
		at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)
		at org.apache.calcite.runtime.Resources$ExInstWithCause.ex(Resources.java:505)
		at org.apache.calcite.runtime.Resources$ExInst.ex(Resources.java:599)
		... 73 more
	Suppressed: java.lang.NullPointerException
		at com.hazelcast.jet.sql_nightly.SqlSTSInnerEquiJoinFaultToleranceStressTest.after(SqlSTSInnerEquiJoinFaultToleranceStressTest.java:164)
		at java.base/java.lang.reflect.Method.invoke(Method.java:566)
		at com.hazelcast.test.AbstractHazelcastClassRunner$ThreadDumpAwareRunAfters.evaluate(AbstractHazelcastClassRunner.java:381)
		at com.hazelcast.test.DumpBuildInfoOnFailureRule$1.evaluate(DumpBuildInfoOnFailureRule.java:37)
		at com.hazelcast.test.jitter.JitterRule$1.evaluate(JitterRule.java:104)
		at com.hazelcast.test.metrics.MetricsRule$1.evaluate(MetricsRule.java:63)
		... 6 more

```
</details>



Standard output can be found here - https://s3.console.aws.amazon.com/s3/buckets/j-artifacts/Hazelcast-master-OpenJDK11-arm64-nightly/32/

Important note:  during this run was also timeout for https://github.com/hazelcast/hazelcast/issues/24563

Reopening as issue occurred again on: _master_ (commit cee0f6eef8a56950e79e8534543790ff87a00476)
with OpenJDK11 (arm64): https://jenkins.hazelcast.com/job/Hazelcast-master-OpenJDK11-arm64-nightly/32/testReport/com.hazelcast.jet.sql_nightly/SqlSTSInnerNonEquiJoinFaultToleranceStressTest/stressTest_2/
<details><summary>Stacktrace:</summary>

```
org.opentest4j.MultipleFailuresError: 
Multiple Failures (2 failures)
	com.hazelcast.sql.HazelcastSqlException: Object 'source_topic_o_e31848e1_964c_44b9_a276_e6f77f3b3f63' not found, did you forget to CREATE MAPPING?
	java.lang.NullPointerException: <no message>
	at com.hazelcast.test.HazelcastSerialClassRunner.runChild(HazelcastSerialClassRunner.java:50)
	at com.hazelcast.test.HazelcastSerialClassRunner.runChild(HazelcastSerialClassRunner.java:29)
	at com.hazelcast.test.AfterClassesStatement.evaluate(AfterClassesStatement.java:41)
	at com.hazelcast.test.OverridePropertyRule$1.evaluate(OverridePropertyRule.java:66)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.lang.Thread.run(Thread.java:829)
	Suppressed: com.hazelcast.sql.HazelcastSqlException: Object 'source_topic_o_e31848e1_964c_44b9_a276_e6f77f3b3f63' not found, did you forget to CREATE MAPPING?
		at com.hazelcast.sql.impl.CoreQueryUtils.toPublicException(CoreQueryUtils.java:50)
		at com.hazelcast.sql.impl.SqlServiceImpl.execute(SqlServiceImpl.java:221)
		at com.hazelcast.sql.impl.SqlServiceImpl.execute(SqlServiceImpl.java:172)
		at com.hazelcast.sql.impl.SqlServiceImpl.execute(SqlServiceImpl.java:168)
		at com.hazelcast.sql.impl.SqlServiceImpl.execute(SqlServiceImpl.java:164)
		at com.hazelcast.sql.SqlService.execute(SqlService.java:89)
		at com.hazelcast.jet.sql_nightly.SqlSTSInnerEquiJoinFaultToleranceStressTest.setUp(SqlSTSInnerEquiJoinFaultToleranceStressTest.java:149)
		at java.base/java.lang.reflect.Method.invoke(Method.java:566)
		at com.hazelcast.test.AbstractHazelcastClassRunner$ThreadDumpAwareRunAfters.evaluate(AbstractHazelcastClassRunner.java:365)
		at com.hazelcast.test.DumpBuildInfoOnFailureRule$1.evaluate(DumpBuildInfoOnFailureRule.java:37)
		at com.hazelcast.test.jitter.JitterRule$1.evaluate(JitterRule.java:104)
		at com.hazelcast.test.metrics.MetricsRule$1.evaluate(MetricsRule.java:63)
		... 6 more
	Caused by: com.hazelcast.sql.impl.QueryException: Object 'source_topic_o_e31848e1_964c_44b9_a276_e6f77f3b3f63' not found, did you forget to CREATE MAPPING?
		at com.hazelcast.sql.impl.QueryException.error(QueryException.java:73)
		at com.hazelcast.jet.sql.impl.validate.HazelcastSqlValidator.newValidationError(HazelcastSqlValidator.java:542)
		at org.apache.calcite.sql.validate.IdentifierNamespace.resolveImpl(IdentifierNamespace.java:183)
		at org.apache.calcite.sql.validate.IdentifierNamespace.validateImpl(IdentifierNamespace.java:188)
		at org.apache.calcite.sql.validate.AbstractNamespace.validate(AbstractNamespace.java:88)
		at org.apache.calcite.sql.validate.SqlValidatorImpl.validateNamespace(SqlValidatorImpl.java:1144)
		at org.apache.calcite.sql.validate.SqlValidatorImpl.validateQuery(SqlValidatorImpl.java:1115)
		at com.hazelcast.jet.sql.impl.validate.HazelcastSqlValidator.validateQuery(HazelcastSqlValidator.java:183)
		at org.apache.calcite.sql.validate.SqlValidatorImpl.validateFrom(SqlValidatorImpl.java:3456)
		at org.apache.calcite.sql.validate.SqlValidatorImpl.validateFrom(SqlValidatorImpl.java:3435)
		at org.apache.calcite.sql.validate.SqlValidatorImpl.validateSelect(SqlValidatorImpl.java:3792)
		at org.apache.calcite.sql.validate.SelectNamespace.validateImpl(SelectNamespace.java:61)
		at org.apache.calcite.sql.validate.AbstractNamespace.validate(AbstractNamespace.java:88)
		at org.apache.calcite.sql.validate.SqlValidatorImpl.validateNamespace(SqlValidatorImpl.java:1144)
		at org.apache.calcite.sql.validate.AbstractNamespace.getRowType(AbstractNamespace.java:119)
		at org.apache.calcite.sql.validate.SelectNamespace.getRowType(SelectNamespace.java:32)
		at org.apache.calcite.sql.validate.AbstractNamespace.getType(AbstractNamespace.java:130)
		at org.apache.calcite.sql.validate.SelectNamespace.getType(SelectNamespace.java:32)
		at org.apache.calcite.sql.validate.SqlValidatorImpl.deriveType(SqlValidatorImpl.java:1899)
		at org.apache.calcite.sql.SqlOperator.deriveOperandType(SqlOperator.java:687)
		at org.apache.calcite.sql.SqlOperator.constructArgTypeList(SqlOperator.java:677)
		at org.apache.calcite.sql.SqlFunction.deriveType(SqlFunction.java:250)
		at org.apache.calcite.sql.SqlFunction.deriveType(SqlFunction.java:231)
		at org.apache.calcite.sql.validate.SqlValidatorImpl$DeriveTypeVisitor.visit(SqlValidatorImpl.java:6534)
		at org.apache.calcite.sql.validate.SqlValidatorImpl$DeriveTypeVisitor.visit(SqlValidatorImpl.java:6521)
		at org.apache.calcite.sql.SqlCall.accept(SqlCall.java:166)
		at org.apache.calcite.sql.validate.SqlValidatorImpl.deriveTypeImpl(SqlValidatorImpl.java:1916)
		at org.apache.calcite.sql.validate.SqlValidatorImplBridge.deriveTypeImpl(SqlValidatorImplBridge.java:40)
		at com.hazelcast.jet.sql.impl.validate.HazelcastSqlValidator.deriveTypeImpl(HazelcastSqlValidator.java:399)
		at org.apache.calcite.sql.validate.ProcedureNamespace.validateImpl(ProcedureNamespace.java:59)
		at org.apache.calcite.sql.validate.AbstractNamespace.validate(AbstractNamespace.java:88)
		at org.apache.calcite.sql.validate.SqlValidatorImpl.validateNamespace(SqlValidatorImpl.java:1144)
		at org.apache.calcite.sql.validate.SqlValidatorImpl.validateQuery(SqlValidatorImpl.java:1115)
		at com.hazelcast.jet.sql.impl.validate.HazelcastSqlValidator.validateQuery(HazelcastSqlValidator.java:183)
		at org.apache.calcite.sql.validate.SqlValidatorImpl.validateTableFunction(SqlValidatorImpl.java:3506)
		at org.apache.calcite.sql.validate.SqlValidatorImpl.validateFrom(SqlValidatorImpl.java:3453)
		at org.apache.calcite.sql.validate.SqlValidatorImpl.validateFrom(SqlValidatorImpl.java:3435)
		at org.apache.calcite.sql.validate.SqlValidatorImpl.validateSelect(SqlValidatorImpl.java:3792)
		at org.apache.calcite.sql.validate.SelectNamespace.validateImpl(SelectNamespace.java:61)
		at org.apache.calcite.sql.validate.AbstractNamespace.validate(AbstractNamespace.java:88)
		at org.apache.calcite.sql.validate.SqlValidatorImpl.validateNamespace(SqlValidatorImpl.java:1144)
		at org.apache.calcite.sql.validate.SqlValidatorImpl.validateQuery(SqlValidatorImpl.java:1115)
		at com.hazelcast.jet.sql.impl.validate.HazelcastSqlValidator.validateQuery(HazelcastSqlValidator.java:183)
		at org.apache.calcite.sql.SqlSelect.validate(SqlSelect.java:282)
		at org.apache.calcite.sql.validate.SqlValidatorImpl.validateScopedExpression(SqlValidatorImpl.java:1090)
		at org.apache.calcite.sql.validate.SqlValidatorImpl.validate(SqlValidatorImpl.java:796)
		at com.hazelcast.jet.sql.impl.validate.HazelcastSqlValidator.validate(HazelcastSqlValidator.java:178)
		at com.hazelcast.jet.sql.impl.parse.SqlCreateView.validate(SqlCreateView.java:104)
		at com.hazelcast.jet.sql.impl.validate.HazelcastSqlValidator.validate(HazelcastSqlValidator.java:146)
		at com.hazelcast.jet.sql.impl.parse.QueryParser.parse0(QueryParser.java:80)
		at com.hazelcast.jet.sql.impl.parse.QueryParser.parse(QueryParser.java:56)
		at com.hazelcast.jet.sql.impl.OptimizerContext.parse(OptimizerContext.java:157)
		at com.hazelcast.jet.sql.impl.CalciteSqlOptimizer.prepare(CalciteSqlOptimizer.java:325)
		at com.hazelcast.sql.impl.SqlServiceImpl.prepare(SqlServiceImpl.java:280)
		at com.hazelcast.sql.impl.SqlServiceImpl.query0(SqlServiceImpl.java:259)
		at com.hazelcast.sql.impl.SqlServiceImpl.execute(SqlServiceImpl.java:204)
		... 16 more
	Caused by: org.apache.calcite.runtime.CalciteContextException: From line 1, column 58 to line 1, column 108: Object 'source_topic_o_e31848e1_964c_44b9_a276_e6f77f3b3f63' not found, did you forget to CREATE MAPPING?
		at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)
		at org.apache.calcite.runtime.Resources$ExInstWithCause.ex(Resources.java:505)
		at org.apache.calcite.sql.SqlUtil.newContextException(SqlUtil.java:948)
		at org.apache.calcite.sql.SqlUtil.newContextException(SqlUtil.java:933)
		at com.hazelcast.jet.sql.impl.validate.HazelcastSqlValidator.newValidationError(HazelcastSqlValidator.java:535)
		... 70 more
	Caused by: org.apache.calcite.sql.validate.SqlValidatorException: Object 'source_topic_o_e31848e1_964c_44b9_a276_e6f77f3b3f63' not found, did you forget to CREATE MAPPING?
		at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)
		at org.apache.calcite.runtime.Resources$ExInstWithCause.ex(Resources.java:505)
		at org.apache.calcite.runtime.Resources$ExInst.ex(Resources.java:599)
		... 73 more
	Suppressed: java.lang.NullPointerException
		at com.hazelcast.jet.sql_nightly.SqlSTSInnerEquiJoinFaultToleranceStressTest.after(SqlSTSInnerEquiJoinFaultToleranceStressTest.java:164)
		at java.base/java.lang.reflect.Method.invoke(Method.java:566)
		at com.hazelcast.test.AbstractHazelcastClassRunner$ThreadDumpAwareRunAfters.evaluate(AbstractHazelcastClassRunner.java:381)
		at com.hazelcast.test.DumpBuildInfoOnFailureRule$1.evaluate(DumpBuildInfoOnFailureRule.java:37)
		at com.hazelcast.test.jitter.JitterRule$1.evaluate(JitterRule.java:104)
		at com.hazelcast.test.metrics.MetricsRule$1.evaluate(MetricsRule.java:63)
		... 6 more

```
</details>



Standard output can be found here - https://s3.console.aws.amazon.com/s3/buckets/j-artifacts/Hazelcast-master-OpenJDK11-arm64-nightly/32/

Important note:  during this run was also timeout for https://github.com/hazelcast/hazelcast/issues/24563

--- a/extensions/mapstore/src/test/java/com/hazelcast/mapstore/GenericMapStoreIntegrationTest.java
+++ b/extensions/mapstore/src/test/java/com/hazelcast/mapstore/GenericMapStoreIntegrationTest.java
@@ -395,7 +395,7 @@ public class GenericMapStoreIntegrationTest extends JdbcSqlTestSupport {
     /**
      * Regression test for https://github.com/hazelcast/hazelcast/issues/22567
      */
-    @Test(timeout = 90_000L)
+    @Test(timeout = 180_000L)
     @Category(NightlyTest.class)
     public void testClear() {
         HazelcastInstance client = client();

